internal/components.go

```diff
@@ -481,13 +486,26 @@ type BuilderHubMockProxy struct {
 
 func (b *BuilderHubMockProxy) Run(service *service, ctx *ExContext) {
 	service.
-		WithImage("docker.io/flashbots/builder-hub-mock-proxy").
-		WithTag("latest").
-		WithPort("http", 8888)
-
-	if b.TargetService != "" {
-		service.DependsOnHealthy(b.TargetService)
-	}
+		WithImage("nginx").
+		WithTag("1.27").
+		WithPort("http", 8888).
+		DependsOnRunning(b.TargetService).
+		WithEntrypoint("/bin/sh").
+		WithArgs("-c", fmt.Sprintf(`cat > /etc/nginx/conf.d/default.conf << 'EOF'
```

Comment:
Why do we need nginx?

Reply:
it's for passing the fake attestation headers to builderhub. BuilderNet uses a proxy that does the TEE attestation and passes these on to BuilderHub.